invenio_rdm_records/services/communities/service.py

```diff
@@ -24,6 +25,38 @@ def create(self, identity, data, record, uow=None):
         pass
 
     @unit_of_work()
-    def delete(self, identity, id_, revision_id=None, uow=None):
+    def delete(self, identity, record_id, data, revision_id=None, uow=None):
         """Remove communities from the record."""
-        pass
+        record = self.record_cls.pid.resolve(record_id)
+        self.require_permission(identity, "remove_community_from_record", record=record)
+
+        valid_data, errors = self.schema.load(
+            data,
+            context={"identity": identity},
+            raise_errors=True,
+        )
+
+        removed_communities = []
+        errored_communities = []
+        community_ids = valid_data["communities"]
+        for community_id in community_ids:
```

Comment:
Shall we put a `max` n. of communities in that list?
If the user passes a high number, the request could timeout.

What is a sensible number here?
If the user provides something higher, we should return `400` and a msg.

Reply:
As a safety measure we could do that (I think 10 max, i.e. a full page size in a search view might be good).

From my basic understanding of the code though I don't see that each loop is that heavy, so it could be fine as well to not cap it.